pkg/relayer/cmd/cmd.go

```diff
@@ -0,0 +1,315 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/url"
+
+	"cosmossdk.io/depinject"
+	cosmosclient "github.com/cosmos/cosmos-sdk/client"
+	cosmosflags "github.com/cosmos/cosmos-sdk/client/flags"
+	cosmostx "github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+	"github.com/pokt-network/poktroll/cmd/signals"
+	"github.com/pokt-network/poktroll/pkg/client/block"
+	eventsquery "github.com/pokt-network/poktroll/pkg/client/events_query"
+	"github.com/pokt-network/poktroll/pkg/client/supplier"
+	"github.com/pokt-network/poktroll/pkg/client/tx"
+	"github.com/pokt-network/poktroll/pkg/relayer"
+	"github.com/pokt-network/poktroll/pkg/relayer/miner"
+	"github.com/pokt-network/poktroll/pkg/relayer/proxy"
+	"github.com/pokt-network/poktroll/pkg/relayer/session"
+)
+
+var (
+	flagSigningKeyName string
+	flagSmtStorePath   string
+	flagSequencerNode  string
+	flagPocketNode     string
+)
+
+func RelayerCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "relayminer",
+		Short: "Run a relay miner",
+		// TODO_TECHDEBT: add a longer long description.
+		Long: `Run a relay miner`,
+		RunE: runRelayer,
+	}
+
+	cmd.Flags().String(cosmosflags.FlagKeyringBackend, "", "Select keyring's backend (os|file|kwallet|pass|test)")
```

Comment:
NIT/ASK: Could we add these `cosmosflags` to the `appGateServer/cmd/cmd.go` file as well for more consistency. I think we rolled our own of these there